single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultLabelProvider.java

```diff
@@ -163,6 +164,14 @@ public List<ElectricalNodeInfo> getElectricalNodesInfos(VoltageLevelGraph graph)
                 .collect(Collectors.toList());
     }
 
+    @Override
+    public List<NodeLegend> getElectricalNodeInfoNodes(ElectricalNodeInfo nodeInfo) {
```

Comment:
```suggestion
    private List<Caption> getCaptions(Bus bus) {
```

Reply:
I could not do that easily without calling LabelProvider twice, so I've done something something slightly different which should satisfy all the other requirements stated in the other comments